src/vcpkg/build.cpp

```diff
@@ -973,8 +973,28 @@ namespace vcpkg
         }
 
         const BuildInfo build_info = read_build_info(fs, paths.build_info_file_path(action.spec));
-        const size_t error_count =
-            perform_post_build_lint_checks(action.spec, paths, pre_build_info, build_info, scfl.source_location);
+        const size_t error_count = [&]() {
+            struct FileSink : MessageSink
+            {
+                WriteFilePointer out_file;
+                StringView stdoutlog;
+                FileSink(Filesystem& fs, StringView stdoutlog)
+                    : out_file(fs.open_for_appending(stdoutlog, VCPKG_LINE_INFO)), stdoutlog(stdoutlog)
```

Comment:
```suggestion
                    : m_stdoutlog(stdoutlog), m_out_file(fs.open_for_appending(m_stdoutlog, VCPKG_LINE_INFO))
```

We had to construct a Path anyway here. (This does require flipping the order in which the members are declared)